server/validate/email/validate.go

```diff
@@ -169,7 +196,15 @@ func (v *validator) Init(jsonconf string) error {
 
 	// Enable auth if login is provided.
 	if v.Login != "" {
-		v.auth = smtp.PlainAuth("", v.Login, v.SenderPassword, v.SMTPAddr)
+		mechanism := strings.ToLower(v.Mechanism)
+		switch mechanism {
+		case "md5":
+			v.auth = smtp.CRAMMD5Auth(v.Login, v.SenderPassword)
+		case "login":
+			v.auth = LoginAuth(v.Login, v.SenderPassword)
+		default:
```

Comment:
Actually, this is not a good idea as it would lead to strange behavior. Please change it to 
```go
case "", "plain":
    v.auth = smtp.PlainAuth("", v.Login, v.SenderPassword, v.SMTPAddr)
default:
    ... fail with an error ...
```

Reply:
I will change with

```go
case "", "plain":
	v.auth = smtp.PlainAuth("", v.Login, v.SenderPassword, v.SMTPAddr)
default:
	return errors.New("Bad Auth Mechanism for SMTP")
```